doc/code/qml_bose.rst

```diff
@@ -10,7 +10,7 @@ This module contains functions and classes for creating and manipulating bosonic
 BoseWord and BoseSentence
 ---------------------------
 
-.. currentmodule:: pennylane.bose
+.. currentmodule:: pennylane
```

Comment:
Looks like we still have duplicates for
`qml.binary_mapping`
`qml.unary_mapping`
`qml.christiansen_mapping`

Clicking the links to these in [the bose module](https://xanaduai-pennylane--7453.com.readthedocs.build/en/7453/code/qml_bose.html) links to `qml.bose.something_mapping` instead of `qml.something_mapping`